File: backend/cricket/urls.py
from django.conf.urls import patterns, url
from cricket.views import *

urlpatterns = patterns(
    "cricket.urls",
    url(r'^add-teams$', home, name='home'),
    url(r'^teams', TeamsDashboard.as_view()),
    url(r'^players', PlayersDashboard.as_view()),
    url(r'^matches', MatchesDashboard.as_view()),
    url(r'^points-table', PointsDashboard.as_view())
)

File: angular/src/app/urls.ts
import { environment } from './../environments/environment';


export class Urls {

    public static readonly TEAMS = environment.apiHostUrl + '/cricket/teams';
    public static readonly PLAYERS = environment.apiHostUrl + '/cricket/players';
    public static readonly MATCHES = environment.apiHostUrl + '/cricket/matches';
    public static readonly POINTS = environment.apiHostUrl + '/cricket/points-table';

}
